bin/elasticurl/main.c

```diff
@@ -721,20 +721,30 @@ int main(int argc, char **argv) {
         .keep_alive_interval_sec = 0,
     };
 
+    struct aws_http2_setting settings_array[] = {
+        {.id = AWS_HTTP2_SETTINGS_ENABLE_PUSH, .value = 0},
+    };
+
+    struct aws_http2_connection_options http2_options = {
+        .initial_settings_array = settings_array,
+        .num_initial_settings = AWS_ARRAY_SIZE(settings_array),
+        .max_closed_streams = AWS_HTTP2_DEFAULT_MAX_CLOSED_STREAMS,
+    };
```

Comment:
trivial/arbitrary: let's just leave this all out and keep elasticurl simple, don't even bother setting `aws_http_client_connection_options.http2_options`

Reply:
I don't get it, actually. Will it be clearer to show the use case of http2_options in elasticurl?